src/vivarium/component.py

```diff
@@ -579,7 +579,14 @@ def get_initialization_parameters(self) -> Dict[str, Any]:
         }
 
     def create_lookup_tables(self, builder: "Builder") -> None:
-
+        """
+        Method to create standard lookup tables for the component. This will create a
+        lookup table for each lookup key in self.standard_lookup_tables property. If
+        there is desire to add additional lookup tables, users can override this method
+        by calling the super method and adding additional lookup tables or overriding the
```

Comment:
This last sentence is difficult to read b/c of lots of "lookup tables". Maybe:
`If there is desired to add additional lookup tables, users have two options: (1) override this method by calling the super method and adding them, or (2) overriding the standard 'lookup_tables' property`

Or something like that.

Reply:
I like this.